### PR TITLE
Optional braces for model is

### LINKED ISSUE #645: Allow omitting braces from `model is`
I feel like the braces shouldn't be needed when such a prevalent usecase is just adding decorators and not adding any new properties.

### PR DESCRIPTION
**Pull request by bterlson**
Fixes #645.

### CODE REVIEW

**Comment by bterlson** [comment]
Indeed!

File: packages/spec/src/spec.emu.html [hunk 1]
```diff
@@ -283,12 +283,15 @@ <h1>Syntactic Grammar</h1>
     `using` IdentifierOrMemberExpression `;`
 
 ModelStatement :
+    DecoratorList? `model` Identifier TemplateParameters? IsModelHeritage `;`
     DecoratorList? `model` Identifier TemplateParameters? ModelHeritage? `{` ModelBody? `}`
```
Comment (nguerrera) [inline packages/spec/src/spec.emu.html:287]: Don't you still need to say that ModelHeritage is either IsModelHeritage or ExtendsModelHeritage somewhere?

Comment (nguerrera) [reply packages/spec/src/spec.emu.html:287]: I thought the spec generation used to fail on a missing reference, but it's green now.

File: packages/compiler/test/parser.test.ts [hunk 2]
```diff
@@ -105,6 +107,9 @@ describe("compiler: syntax", () => {
         [/Cannot use default with non optional properties/],
       ],
       ["model", [/Identifier expected/]],
+      ["model Car is vehicle", [/'{' expected/]],
```
Comment (nguerrera) [inline packages/compiler/test/parser.test.ts:110]: should this be `;` or `{` expected?

File: packages/spec/src/spec.emu.html (hunk 1 above)
Comment (nguerrera) [reply packages/spec/src/spec.emu.html:287]: Really? What about `model X is Y { ... }`. That's still valid, right?

Comment (bterlson) [reply packages/spec/src/spec.emu.html:287]: Hahah you are right this change is nonsense.

File: packages/compiler/test/parser.test.ts (hunk 2 above)
Comment (bterlson) [reply packages/compiler/test/parser.test.ts:110]: hmm, I suppose so...

Comment (nguerrera) [reply packages/compiler/test/parser.test.ts:110]: Not that concerned if that's too much trobule.

**Review by nguerrera — APPROVED** [review]

**Pull request merged**